tests/test_state.py

```diff
@@ -269,7 +269,8 @@ def test_fp_state():
                 "length": 241,
                 "method": "fcfp-count",
                 "counting": True,
-                "nBits": 241,
+                "fpSize": 241,
+                "atomInvariantsGenerator": "SerializableMorganFeatureAtomInvGen",
```

Comment:
Thank you !

Reply:
I have been a bit busy so could not finish the PR,, but I hope this can get merged soon.